homeassistant/components/onvif/device.py

```diff
@@ -242,9 +242,13 @@ async def async_get_capabilities(self):
 
         pullpoint = False
         try:
-            event_service = self.device.create_events_service()
-            event_capabilities = await event_service.GetServiceCapabilities()
-            pullpoint = event_capabilities and event_capabilities.WSPullPointSupport
+            self.device.get_definition("pullpoint")
+            await self.device.create_pullpoint_subscription()
```

Comment:
A problem with this approach is that we are now creating two subscription services.  The one that gets created here is subsequently abandoned and not closed.  Cameras have a limited number of subscriptions available, and we are now taking one for the duration of the timeout on the device. If this is done too many times, the device may not have any subscriptions left by the time we are ready to actually start polling for events.

Reply:
I thought about it, but i doesn't thought that was a problem, because I believed that the pulpoint subscription had a time limit and once reached it close itself. I check if there is a way to manually close subscription